docs/index.rst

```diff
@@ -108,6 +108,8 @@ Mission Statement
     development/index
     CHANGELOG.md
     roadmap
+    building_local_docs.rst
+    coding_guidelines.rst
```

Comment:
```suggestion
```
You probably didn't want them in this PR - remove them since even mentioned files are not here. :smile: